packages/manager/apps/dedicated/client/app/dedicated/server/details/server.controller.js

```diff
@@ -216,7 +216,19 @@ export default class ServerCtrl {
       }
     };
 
-    this.$scope.$on('dedicated.informations.reload', () => {
+    this.$scope.$on('dedicated.informations.reload', (e, params) => {
+      if (
+        params &&
+        Object.prototype.hasOwnProperty.call(params, 'monitoring')
+      ) {
+        this.server.monitored = params.monitoring;
```

Comment:
No other way to reload ? Could it be achieved by reloading the state when coming back to the dashboard if we are in another state ?

Reply:
Reloading the route state will be too costly. It will reinvoke all API calls. The only thing that needs to be updated is the monitoring state (true or false). This value is available in the child route and needs to be passed back.